app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
```

Comment:
you need write without brackets, but its not big issue

Reply:
yeah, but the big issue are line breaks:)

Reply:
I have some issue with PyCharm, spent a lot of time with is, will ask someone tomorrow what i need to do with is, probably during QnA.
Please check second commit (posted after first one).

Reply:
issue resolved, i had an issue with synchronization of local and remote branches
please check last commit, it is final solution